packages/perseus/src/widgets/orderer/score-orderer.test.ts

```diff
@@ -56,20 +56,4 @@ describe("ordererValiator", () => {
         // assert
         expect(result).toHaveBeenAnsweredIncorrectly();
     });
-
-    it("is invalid when the when the user has not started ordering the options and current is empty", () => {
```

Comment:
I think it is worth retaining a unit test for the scorer that makes sure the score is `"invalid"` for this case. This ensures we're correctly calling and handling the result of the validator in the scorer.

Reply:
Added this back :) Do you think all scoring functions should retain a test confirming it can handle validation results? Also, I remember hearing about how it's ideal to make sure tests don't overlap so that multiple tests don't fail as a result of one piece of code failing (for example, validation). Is that why you mocked this out in your PRs?

Reply:
Updated the test to mock the validation and added a test confirming scoring handles passing validation with validation mocked.